packages/block-library/src/table/deprecated.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText, getColorClassName } from '@wordpress/block-editor';
+
+const blockAttributes = {
+	hasFixedLayout: {
+		type: 'boolean',
+		default: false,
+	},
+	backgroundColor: {
+		type: 'string',
+	},
+	head: {
+		type: 'array',
+		default: [],
+		source: 'query',
+		selector: 'thead tr',
+		query: {
+			cells: {
+				type: 'array',
+				default: [],
+				source: 'query',
+				selector: 'td,th',
+				query: {
+					content: {
+						type: 'string',
+						source: 'html',
+					},
+					tag: {
+						type: 'string',
+						default: 'td',
+						source: 'tag',
+					},
+				},
+			},
+		},
+	},
+	body: {
+		type: 'array',
+		default: [],
+		source: 'query',
+		selector: 'tbody tr',
+		query: {
+			cells: {
+				type: 'array',
+				default: [],
+				source: 'query',
+				selector: 'td,th',
+				query: {
+					content: {
+						type: 'string',
+						source: 'html',
+					},
+					tag: {
+						type: 'string',
+						default: 'td',
+						source: 'tag',
+					},
+				},
+			},
+		},
+	},
+	foot: {
+		type: 'array',
+		default: [],
+		source: 'query',
+		selector: 'tfoot tr',
+		query: {
+			cells: {
+				type: 'array',
+				default: [],
+				source: 'query',
+				selector: 'td,th',
+				query: {
+					content: {
+						type: 'string',
+						source: 'html',
+					},
+					tag: {
+						type: 'string',
+						default: 'td',
+						source: 'tag',
+					},
+				},
+			},
+		},
+	},
+};
+
+const supports = {
+	align: true,
+};
+
+const deprecated = [
```

Comment:
Would be great to add a test fixture for this deprecation. There are some examples of them in the folder with the fixtures (some have `__deprecated` in the filename):
https://github.com/WordPress/gutenberg/tree/master/packages/e2e-tests/fixtures/blocks

It should just be a matter of grabbing the block HTML from `master` (including the comment delimeters) and pasting it into the html file, then regenerating the fixtures. If the deprecation works it'll output a valid block.

Reply:
Done, thanks for the reminder!